kv/union_iter.go

```diff
@@ -47,14 +47,14 @@ func NewUnionIter(dirtyIt Iterator, snapshotIt Iterator, reverse bool) (*UnionIt
 	return it, nil
 }
 
-// Go next and update valid status.
+// dirtyNext and update valid status.
```

Comment:
cc @winoros PTAL. I am not sure the correctness of comment.

Reply:
Better follow the pattern `FuncName does something`

Reply:
@zhexuany @winoros PTAL again